### PR TITLE
velero-plugin-for-csi initial setup

### PR DESCRIPTION
**Pull request by hhpatel14**

### CODE REVIEW

**Comment by weshayutin** [comment]
which of the checks is running the unit tests?

File: ci-operator/config/openshift/velero-plugin-for-csi/openshift-velero-plugin-for-csi-main.yaml
Comment (danilo-gemoli) [inline ci-operator/config/openshift/velero-plugin-for-csi/openshift-velero-plugin-for-csi-main.yaml:19]: From my understanding you're claiming a release that you don't use. Is it intentional?

Comment (hhpatel14) [reply ci-operator/config/openshift/velero-plugin-for-csi/openshift-velero-plugin-for-csi-main.yaml:19]: This is intentional for now.

Comment (hhpatel14) [reply ci-operator/config/openshift/velero-plugin-for-csi/openshift-velero-plugin-for-csi-main.yaml:19]: @danilo-gemoli Question: I have unit test configured in this file. Should I expect that running in the checks below?

**Comment by hhpatel14** [comment]
presubmit

**Comment by hhpatel14** [comment]
/pj-rehearse pull-ci-openshift-velero-plugin-for-csi-main-unit-test

**Comment by hhpatel14** [comment]
/pj-rehearse pull-ci-openshift-velero-plugin-for-csi-main-unit-test

**Comment by hhpatel14** [comment]
/pj-rehearse pull-ci-openshift-velero-plugin-for-csi-main-unit-test

**Comment by hhpatel14** [comment]
/pj-rehearse pull-ci-openshift-velero-plugin-for-csi-main-unit-test

**Comment by weshayutin** [comment]
--- PASS: TestSetVolumeSnapshotContentDeletionPolicy (0.00s)
    --- PASS: TestSetVolumeSnapshotContentDeletionPolicy/should_update_DeletionPolicy_of_a_VSC_from_retain_to_delete (0.00s)
    --- PASS: TestSetVolumeSnapshotContentDeletionPolicy/should_be_a_no-op_updating_if_DeletionPolicy_of_a_VSC_is_already_Delete (0.00s)
    --- PASS: TestSetVolumeSnapshotContentDeletionPolicy/should_update_DeletionPolicy_of_a_VSC_with_no_DeletionPolicy (0.00s)
    --- PASS: TestSetVolumeSnapshotContentDeletionPolicy/should_return_not_found_error_if_supplied_VSC_does_not_exist (0.00s)
=== RUN   TestIsByBackup
--- PASS: TestIsByBackup (0.00s)
PASS
ok  	github.com/vmware-tanzu/velero-plugin-for-csi/internal/util	0.032s

Unit tests look good.
I don't see where the build image is pushed to quay.  Can you point that out to me please.  It would be cool if the local image sha was logged so we can ensure the built image made it to quay.

**Comment by danilo-gemoli** [comment]
it's ok for me.
/lgtm
/hold
unblock this PR whenever you feel confident enough

**Review by kaovilai — APPROVED** [review]

**Pull request merged**
